### ISSUE TITLE
Clarify how to get starting drawing something

### ISSUE THREAD

**Issue by uvtc** [quil/quil-site#15]
For running quil on Clojure, the quil.info webpage says, "3. Open src/my_sketch/core.clj, and evaluate it.", but it's not clear what "evaluate it" means.

I've run `lein new quil my-art` and `cd my-art`. I look at src/my_art/core.clj. It contains functions `setup`, `update-state`, and `draw-state`. It also has a `q/defsketch`. I run `lein repl`, but nothing happens.

How do I get that code to run so a drawing appears? The website should explicitly explain how to do that.

**Comment by uvtc** [comment]
Ok. I now see that I can run `(use 'my-art.core)` and the window pops up running the demo code.

If I then edit the demo code, and run `(use :reload 'my-art.core)`, I see that a 2nd window pops up showing my changes. So, with this simple workflow, looks like I need to:

 1. edit code
 2. close existing window
 3. `(use :reload 'my-art.core)`, and window comes up again with my changes.

**Comment by nbeloglazov** [comment]
Added link to more detailed tutorial. Hope this'll make it easier to start.

**Comment by vsvasya** [comment]
It is also very unclear what to do if nothing happens :)
After carefully doing all steps of tutorial repl command just returns nil, and no graphics windows appears.

my-art.core=> (use :reload 'my-art.core)
nil

clojure 1.9.0, quil 2.7.1, java-8-openjdk-amd64, Ubuntu 17.04

**Comment by nbeloglazov** [comment]
Have you created my-art using `lein new quil my-run`? If so - yes, there is a bug: template was changed but instructions weren't updated. To run the project try the following:
```bash
lein run
```
or 
```clj
my-art.core=> (use :reload 'my-art.core)
my-art.core=> (-main)
```

I should update docs or probably rollback template change so that it works by just reloading namespace.

**Comment by vsvasya** [comment]
Yes, like in tutorial I created project with `lein new quil my-art`.

Both `lein run` and `(-main)` in repl works! Thank you)
